commands/commands.go

```diff
@@ -32,29 +32,29 @@ func newPluginDriver(driverName string, rawContent []byte) (*rpcdriver.RpcClient
 	return d, nil
 }
 
-func fatal(args ...interface{}) {
-	log.Fatal(args...)
+func fatalOnError(command func(context *cli.Context) error) func(context *cli.Context) {
+	return func(context *cli.Context) {
+		if err := command(context); err != nil {
+			fatal(err)
```

Comment:
This can just call `log.Fatal` directly now.  Since the cleanup has been moved to server-side, we can eliminate the `fatal` and `fatalf` wrappers completely.